requirements-dev.txt

```diff
@@ -1,6 +1,11 @@
+black
+mypy
+pre-commit
```

Comment:
If you use pre-commit, you don't need the rest. Pre-commit will install them itself in a virtual environment.

Reply:
Thanks! I did not know. Fixed in 203133b